internal/oci/container.go

```diff
@@ -344,6 +344,23 @@ func (c *Container) exitFilePath() string {
 	return filepath.Join(c.dir, "exit")
 }
 
+// IsRunning is a function that checks if a specific pid is running
+// it is used to check a container state when we don't want (or don't trust) a `$runtime state` call
+func (c *Container) IsRunning() bool {
+	process, err := os.FindProcess(c.state.Pid)
```

Comment:
On UNIX, os.FindProcess always succeeds (see its doc), so you safely simplify by doing
```go
  // On Unix systems, FindProcess always succeeds
	process, _ := os.FindProcess(c.state.Pid)
```
and dropping the error check below

Reply:
...unless this code is platform-agnostic, of course

Reply:
this one is still not addressed. Not a big issue but would save a few lines code (which is always good)